projects/v1/default.wmf.yaml

```diff
@@ -67,7 +67,7 @@ paths:
       - path: v1/related.js
         options: '{{options.related}}'
       - path: v1/random.yaml
-        options: '{{merge({"random_cache_control": "s-maxage=2, max-age=1"}, options.wikifeeds)}}'
+        options: '{{options.wikifeeds)}}'
```

Comment:
extra `)` at the end there. Same with the others below

Reply:
Hi, @clarakosi , thanks! ")" removed.